### PR TITLE
📝Add intro to Rules to Better LINQ

### PR DESCRIPTION
**Pull request by AlexRothwell**

### CODE REVIEW

File: categories/software-development/rules-to-better-linq.md [hunk 1]
```diff
@@ -11,3 +11,12 @@ index:
 - how-linq-has-evolved
 
 ---
+Language Integrated Query or LINQ for short, is described by Microsoft as "a set of technologies based on the integration of query capabilities directly into the C# language".
+When used right, it can result in some very powerful and readable code, delivering value and solutions faster.
+
+LINQ is an important part of the .NET ecosystem, regularly receiving large improvements.
+
+`youtube: https://www.youtube.com/watch?v=zCKwlgtVLnQ`
+**Video: [Nick Chapsas](https://www.youtube.com/@nickchapsas) summarised the upgrades included in .NET 7 (12 min)**
```
Comment (GordonBeeming) [inline categories/software-development/rules-to-better-linq.md:20]: @AlexRothwell 

For video figures we use the Video title, Tip 5 on https://ssw.com.au/rules/add-useful-and-concise-figure-captions/#tip-5-add-the-titles--length-on-video-captions :)

For time, we always round down, your format is correct, see https://ssw.com.au/rules/right-format-to-write-videos-time-length/

Comment (AlexRothwell) [reply categories/software-development/rules-to-better-linq.md:20]: @GordonBeeming Thanks for the catch on the length!

I'm don't agree on using the video's title for the figure in all cases. Here, it's particularly clickbaity and may not actually fully convey what the video covers. Also, creators can change the titles whenever they like so there's not guarantee it'd match anyway

Comment (GordonBeeming) [reply categories/software-development/rules-to-better-linq.md:20]: Generally we'd put a bit before the video with what you've outlines here saying in the below video Nick summarised the upgrades included in .NET 7. That then gives the context of what you about to see.

In saying that, I get the clickbait titles and that they can be changed, the original intent around using the same name for the figure is that if the video no longer exists (from the embed) we know what we're looking for to replace it... maybe there was a re-upload, and without the title we'd not know what we looking for 🙂.

It is a good conversation to have though, i.e.: when does it make sense not to use the video title, and then we update the rule with the outcome of that conversation :)

Comment (AlexRothwell) [reply categories/software-development/rules-to-better-linq.md]: Added a sentence before the video and added the title in a comment for tracking

**Review by GordonBeeming — APPROVED** [review]
LGTM

**Pull request merged**